scripts/bandit.sh

```diff
@@ -4,7 +4,12 @@ set -v
 
 echo 'Installing bandit'
 #pip install --user bandit
-pip3 install bandit
+if [ -x "$(command -v pip3)" ]; then
```

Comment:
Actually I think `pip install bandit` is enough

It will be more convincing if you can test allura's bandit job with this branch

Reply:
Agree, but to keep the changes of @everops-alex as it was, I added if condition